playwright/e2e/12-notification-modal.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { expect } from '@playwright/test'
+
+import { test } from '../fixtures'
+test.beforeEach(async ({ layoutPage, loginPage }) => {
+  await layoutPage.gotoHome('dashboard')
+  await loginPage.loginDashboard()
+})
+test.describe('12. TEST Notification Modal ', () => {
+  test('TC-01: should modal message be in the users language', async ({
+    page,
+  }) => {
+    const Subscribe_btn = await page.getByTestId(
+      'button-subscribe-notification',
+    )
+    await expect(Subscribe_btn).toHaveText('Subscribe') // The Turkish message content should contain the word 'Subscribe'
+    const chakra_icon = await page.locator(
+      '.chakra-modal__close-btn.css-1h7f1qa',
+    )
+    await chakra_icon.click()
+    const pageUrl = page.url()
+    await page.goto(pageUrl + `/tr`)
+    await expect(Subscribe_btn).toHaveText('Abone ol') // The Turkish message content should contain the word 'Abone ol'
+    await chakra_icon.click()
+    await page.goto(pageUrl + `/nl`)
+    await expect(Subscribe_btn).toHaveText('Abonneren') // The Turkish message content should contain the word 'Abonneren'
+
+    await chakra_icon.click()
+  })
+  test('TC-02: Should the subscribe and close buttons be visible, both buttons must be visually distinct and easy to find. ', async ({
+    page,
+  }) => {
+    const Subscribe_btn = await page.getByTestId(
+      'button-subscribe-notification',
+    )
+    await expect(Subscribe_btn).toBeVisible() // Subscribe button must be visible
+    const Close_btn = await page.getByTestId('button-close-notification')
+    await expect(Close_btn).toBeVisible() // Close button must be visible
+    const subscribeText = await Subscribe_btn.innerText()
+    const closeText = await Close_btn.innerText()
+    await expect(subscribeText).not.toBe(closeText) // Text content of subscribe and close buttons must be different
+    const subscribeClass = await Subscribe_btn.getAttribute('class')
+    const closeClass = await Close_btn.getAttribute('class')
+    await expect(subscribeClass).not.toBe(closeClass) // Classes of subscribe and close buttons must be different
+  })
+  test('TC-03: Should the Popup Menu be dismissible without subscribing ', async ({
+    page,
+  }) => {
+    const chakra_icon = await page.locator(
+      '.chakra-modal__close-btn.css-1h7f1qa',
+    ) // testid istenmeli
```

Comment:
You can add `// TODO: Request a testid` comment where you need it and mention from the pull request

```suggestion
    // TODO: Request a testid
    const chakra_icon = await page.locator(
      '.chakra-modal__close-btn.css-1h7f1qa',
    )
```

@7alip Please add testid here!

Reply:
Check if `button-close-notification-modal` testid exists.